packages/@tinacms/graphql/src/index.ts

```diff
@@ -40,6 +40,35 @@ import type {
   TinaFieldBase,
 } from './primitives/types'
 
+import { Octokit } from '@octokit/rest'
+
+export const listBranches = async ({auth, owner, repo}) => {
+  const appOctoKit = new Octokit({ auth })
+  const branchList = await appOctoKit.repos.listBranches({
+    owner,
+    repo
+  })
+
+  return branchList
+}
+
+export const createBranch = async ({ auth, owner, repo, name, branch }) => {
```

Comment:
I would suggest carrying over the `branch` -> `baseBranch` name change to this level as well.

Reply:
Makes sense to me, those changes just pushed up.